integrations/extensions/starter-kits/search-with-milvus/search-with-watsonx-data-milvus.md

```diff
@@ -0,0 +1,109 @@
+This is a documentation about how to set up a search integration with the Milvus service from watsonx.data for using search or conversational search feature in watsonx Assistant.
+
+## Before you begin
+1. Provision an watsonx.data instance
+    * watsonx.data on IBM Cloud: https://cloud.ibm.com/watsonxdata
+    * watsonx.data on Cloud Pak for Data (on-prem): follow [this doc](https://www.ibm.com/docs/en/cloud-paks/cp-data/5.0.x?topic=services-watsonxdata) to install and set up watsonx.data on Cloud Pak for Data 
+2. Add an Milvus service in the watsonx.data console
+    * watsonx.data on IBM Cloud: [Adding a Milvus service](https://cloud.ibm.com/docs/watsonxdata?topic=watsonxdata-adding-milvus-service)
+    * watsonx.data on Cloud Pak for Data (on-prem): [Adding a Milvus service](https://www.ibm.com/docs/en/watsonx/watsonxdata/2.0.x?topic=milvus-adding-service)
+
+## Table of contents
+* [Step 1: Collect Milvus connection info](#step-1-collect-milvus-connection-info)
+  * [Get the credentials](#get-the-credentials)
+  * [Get other connection details](#get-other-connection-details)
+* [Step 2: Ingest data into Milvus](#step-2-ingest-data-into-milvus)
+  * [Option 1: Ingest data through watsonx.ai](#option-1-ingest-data-through-watsonxai)
+  * [Option 2: Ingest data using custom code](#option-2-ingest-data-using-custom-code)
+* [Step 3: Connect to watsonx Assistant](#step-3-connect-to-watsonx-assistant)
+  * [Option 1: Use the built-in Milvus search integration](#option-1-use-the-built-in-milvus-search-integration)
+  * [Option 2: Use the Custom Service search](#option-2-use-the-custom-service-search)
+
+## Step 1: Collect Milvus connection info
+### Get the credentials
+#### Username
+The default username is `ibmlhapikey` for the Milvus service on watsonx.data
+
+#### Password
+IBM API key is used as the password for the Milvus service on watsonx.data. There are two options to get the API key to access watsonx.data:
+* **(Option 1) Service ID access**: Assign a watsonx.data access role to a service ID and create a new API key or use an existing one from this service ID. You can manage service IDs from the [IBM Cloud IAM Service IDs page](https://cloud.ibm.com/iam/serviceids). 
+* **(Option 2) User access**: Assign a watsonx.data access role to a user and create a new API key or use an existing one from this user. You can manage user access and roles either from the [IBM Cloud IAM Users page](https://cloud.ibm.com/iam/users) or within the watsonx.data console by following the [Managing user access](https://cloud.ibm.com/docs/watsonxdata?topic=watsonxdata-manage_access) doc.
+
+
+### Get other connection details
+Here are the steps to collect other Milvus connection details from the watsonx.data console:
+
+1. Go to the `Infrastructure manager` page.
+2. Click the Milvus service to open the `Details` page.
+3. Click on `View connect details` to view more connection details.
+4. Collect the service GRPC `host`, `port` and the SSL certificate.
```

Comment:
If I recall, the UI shows more than one SSL certificate.  Which one should they collect?